### ISSUE TITLE
Add a regex search column to PNH for complex name matching

### ISSUE THREAD

**Issue by mapomatic** [WazeUSA/WME-Place-Harmonizer#94]
Regular expressions allow much more advanced pattern matching, and can be used to also exclude similar names when naming collisions would normally occur.

**Comment by StormeHawke** [comment]
*yes*

I actually thought this was already a thing.  Would be awesome to be able to recognize common misspellings (though some kind of confirmation if the regex was used might be good in case of mis-matches).

If you need any help with regexes I became a defacto expert in their use in one of my previous jobs so feel free to ask me if you have trouble with one

**Comment by mapomatic** [comment]
Thanks @StormeHawke!  I've got a little bit of regex experience but certainly wouldn't consider myself an expert.  Will most likely be in contact with you soon ;)

**Comment by StormeHawke** [comment]
yeah, just give me a place name and what variations you want to recognize
and I can build a regex.

Also this is a useful tool for testing them
http://myregexp.com/index.html
